tests/jax/components/training/model_updating_test.py

```diff
@@ -0,0 +1,394 @@
+# python3
+# Copyright 2021 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for MAPGMinibatchUpdate and MAPGEpochUpdate class for Jax-based Mava systems"""
+
+from types import SimpleNamespace
+from typing import Any, Callable, Dict, Tuple
+
+import jax
+import jax.numpy as jnp
+import optax
+import pytest
+
+from mava.components.jax.training import Batch
+from mava.components.jax.training.model_updating import (
+    MAPGEpochUpdate,
+    MAPGMinibatchUpdate,
+)
+from mava.systems.jax.trainer import Trainer
+from mava.types import OLT
+
+
+def fake_grad_fn(
```

Comment:
Can we call this `fake_ppo_grad_fn`, since not algs use advantage.

Reply:
Done!